subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/attributes/ExclusiveVariantsIntegrationTest.groovy

```diff
@@ -45,25 +48,46 @@ class ExclusiveVariantsIntegrationTest extends AbstractIntegrationSpec {
                         attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_RUNTIME))
                         attribute(Category.CATEGORY_ATTRIBUTE, objects.named(Category, "custom"))
                     }
+                    if (!"${capability}".equals('default')) {
+                        outgoing.capability('org.test:sample:1.0')
+                    }
                 }
-            }""".stripIndent()
+
+                consumer {
```

Comment:
Having a configuration named `consumer` but with the `canBeConsumed` flag set to `false` is confusing to me. I would expect this to be called `resolver` instead since `canBeresolved` is set to `true`.